src/main/scala/mesosphere/marathon/MetricsModule.scala

```diff
@@ -0,0 +1,45 @@
+package mesosphere.marathon
+
+import com.codahale.metrics.servlets.MetricsServlet
+import java.lang.management.ManagementFactory
+
+import com.codahale.metrics.MetricRegistry
```

Comment:
So we are using Codahale Metrics for the server but Kamon for the service itself?

Reply:
I second to switch to Kamon here for consistency in the next PR